### PR TITLE
set refresh hint to 1/3 of default CA TTL value fixes #335

### LINKED ISSUE #335: SVID is not valid: public key "X5ZOAszrYj0LnaHdUqRWLZcMtzpgcY9L" not found in trust domain {federated cluster}
When the CA in spire rotates, the federation connections are interrupted. I don't have mitigation steps to reconnect them. Deleting the federation CR and recreating them, does not re-establish trust.

Repo Steps:
1. Set your CA TTL to a short period of time across two clusters ie. 10mins
2. Wait until TTL expires
3. Mint a token on clusterA, attempt to validate it on clusterB

Reported on the old repo here: https://github.com/spiffe/helm-charts/issues/364

Workaround:
Use the spire-server cli to force a refresh
```
kubectl -n spire-server exec -q spire-server-0 -- \
  spire-server federation refresh -id {trust domain of federated server}
```

### PR DESCRIPTION
**Pull request by drewwells**

### CODE REVIEW

File: charts/spire/charts/spire-server/values.yaml [hunk 1]
```diff
@@ -210,6 +210,8 @@ federation:
     port: 8443
     ## @param federation.bundleEndpoint.address Address for trust bundle federation
     address: "0.0.0.0"
+    ## @param federation.bundleEndpoint.refresh_hint Hint used by federated servers on how often to refresh the bundle. CA TTL must be 3-5x the duration of this value to ensure public keys are loaded on federated servers prior to private key rotation on remote server.
+    refresh_hint: "8h"
```
Comment (kfox1111) [inline charts/spire/charts/spire-server/values.yaml:214]: Thinking we choose the same default spire used here? "5m"?

**Review by kfox1111 — COMMENTED** [review]
Generally looks good. one comment inline

File: charts/spire/charts/spire-server/values.yaml (hunk 1 above)
Comment (drewwells) [reply charts/spire/charts/spire-server/values.yaml:214]: Works for me

**Review by faisal-memon — APPROVED** [review]
Thanks @drewwells for this submission

**Review by kfox1111 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**